src/git-town

```diff
@@ -33,4 +33,5 @@ elif [ -n "$1" ]; then
   echo_inline_usage_or "git town main-branch [<branchname>]"
   echo_inline_usage_or "git town non-feature-branches [(--add | --remove) <branchname>]"
   echo_inline_usage_or "git town version"
+  exit_with_error
```

Comment:
Actually, given that we output pretty lengthy documentation, the newline adds some visual space between that documentation block and the next prompt. So I vote for keeping it.

With empty line:

```
~/D/b/git-town [kg-town-error-code] > git town euoeu
error: unsupported subcommand 'euoeu'
usage: git town
   or: git town config
   or: git town help
   or: git town main-branch [<branchname>]
   or: git town non-feature-branches [(--add | --remove) <branchname>]
   or: git town version

~/D/b/git-town [kg-town-error-code] > 
```

Without empty line:

```
error: unsupported subcommand 'euoeu'
usage: git town
   or: git town config
   or: git town help
   or: git town main-branch [<branchname>]
   or: git town non-feature-branches [(--add | --remove) <branchname>]
   or: git town version
~/D/b/git-town [kg-town-error-code] > 
```